test/unit/traversing.js

```diff
@@ -743,6 +743,37 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
```

Comment:
```HTML
<div id="outerBox">
    <template>
        <img src="path/to/image.jpg" onload="alert('OK')" />
    </template>
</div>
```
In my opinion, event handler executes or not, it's depended on browser support of `<template />` during parsing HTML code.

IE treats `<template />` as an `HTMLUnknownElement`, so `<img />` will be in the same document (fragment) of this template, then `onload` will be executed.

Otherwise, `<img />` will be in a `DocumentFragment`, and `onload` won't be executed until `$('#outerBox template').contents().appendTo('#outerBox')`.